src/test/java/com/naturalautomation/selenium/pages/myfiddle/MyFiddlePage.java

```diff
@@ -56,7 +57,6 @@ protected String getURL() {
     @Override
     protected void selectIFrame() {
         getDriver().switchTo().defaultContent();
-        List<WebElement> iframes = getDriver().findElements(By.tagName("iframe"));
-        iframes.stream().findFirst().ifPresent(iframe -> getDriver().switchTo().frame(iframe));
```

Comment:
This function is intended to try to autoselect an iframe if present. Your code will try to select first iframe even if not present.